angular2-jwt.ts

```diff
@@ -304,56 +304,34 @@ export function tokenNotExpired(tokenName = AuthConfigConsts.DEFAULT_TOKEN_NAME,
   return token != null && !jwtHelper.isTokenExpired(token);
 }
 
-export const AUTH_PROVIDERS: Provider[] = [
```

Comment:
You said this was not compatible with AoT. Can you point to documentation about that?